docs/src/building.md

```diff
@@ -344,3 +344,40 @@ Examples of builders that depend on other binaries include:
 * [`Xorg_libX11`](https://github.com/JuliaPackaging/Yggdrasil/blob/eb3728a2303c98519338fe0be370ef299b807e19/X/Xorg_libX11/build_tarballs.jl#L36-L42)
   depends on `Xorg_libxcb_jll`, and `Xorg_xtrans_jll` at build- and run-time,
   and on `Xorg_xorgproto_jll` and `Xorg_util_macros_jll` only at build-time.
+
+# Building and testing JLL packages locally
+
+As a package developer, you may want to test JLL packages locally, or as a binary dependency
+developer you may want to easily use custom binaries.  In both cases, the new `dev` and
+`overrides` directory support make it easy to get complete control over the local JLL package
+state.
+
+## Overriding a prebuilt JLL package's binaries
+
+After running `pkg> dev LibFoo_jll`, a local JLL package will be checked out to your depot's
+`dev` directory, (on most installations this is `~/.julia/dev`) and by default it will make
+use of binaries within your depot's `artifacts` directory.  However, if an `override`
+directory is present within that package directory, it will look within that directory for
```

Comment:
Again, "it".

Also, when reading this I was unsure what "that package directory" refers to -- presumably the one in `~/.julia/dev`? I didn't see mention of another, but "that" sounds (to me, as a non-native speaker, so this may very well be entirely be my own fault!) as if there was some alternate dir to consider? (And perhaps there is and I am just oblivious to it). Perhaps this could be made clearer; I am not quite sure how so; but for me, the following already is a bit clearer.

```suggestion
directory is present within its package directory, the JLL will look within that directory for
```